ui/src/app/shared/components/orb/sink-control/sink-control.component.html

```diff
@@ -1,22 +1,27 @@
-<nb-select #sinkSelLead
-           [(selected)]="selectedSink"
-           appearance="filled"
-           autofocus
-           class="sink-dropdown-selector"
-           data-orb-qa-id="sink_selector"
-           placeholder="Sink Name">
-  <nb-option *ngFor="let sink of availableSinks"
-             [attr.data-orb-qa-id]="'available_sink: '+sink.id" [id]="sink.id"
-             [value]="sink">{{ sink.name }}</nb-option>
+<nb-select
+    #sinkSelLead
+    [(selected)]="selectedSink"
+    appearance="filled"
+    autofocus
+    class="sink-dropdown-selector"
+    data-orb-qa-id="sink_selector"
+    placeholder="Sink Name">
+  <nb-option
+    *ngFor="let sink of sinksList"
+    [attr.data-orb-qa-id]="'available_sink: '+sink.id"
+    [id]="sink.id"
+    [value]="sink">{{ sink.name }}</nb-option>
 </nb-select>
-<button (click)="onAddSink(selectedSink)"
-        [disabled]="!selectedSink"
-        data-orb-qa-id="addSink"
-        ghost
-        nbButton>
-  <nb-icon icon="plus-outline"
-           size="14"
-           status="primary"
-           style="color: #df316f;">
+<button
+  (click)="onAddSink(selectedSink)"
+  [disabled]="!selectedSink"
+  data-orb-qa-id="addSink"
+  ghost
+  nbButton>
+  <nb-icon
+    icon="plus-outline"
+    size="14"
+    status="primary"
+    style="color: #df316f;">
   </nb-icon>
 </button>
```

Comment:
empty line at end of file, you want to get rid of this, set up this in your editor
![Screenshot from 2022-05-04 11-33-22](https://user-images.githubusercontent.com/6593889/166704548-65363704-3ef4-4b02-b82e-72f8003ca589.png)

Reply:
In vs code you can use in the settings.json too
```json
"files.trimTrailingWhitespace": true
"files.trimFinalNewlines": true
"files.insertFinalNewline": true
```

Reply:
@lpegoraro interesting. I believe Intellij may have read this rule from existing codebase - mainflux.

I usually forget to leave those newlines in newly created files and get the complain, hence it is in my autoformat to leave one empty line.

I can update this rule and perhaps wait to apply it to all documents on the beginning of the next sprint.